libs/datamodel/core/src/transform/ast_to_dml/validation_pipeline.rs

```diff
@@ -64,11 +67,17 @@ impl<'a, 'b> ValidationPipeline<'a> {
         }
 
         // TODO: Move consistency stuff into different module.
-        // Phase 5: Consistency fixes. These don't fail.
-        if let Err(mut err) = self.standardiser.standardise(ast_schema, &mut schema) {
+        // Phase 5: Consistency fixes. These don't fail and always run, during parsing AND formatting
+        if let Err(mut err) = self.standardiser_for_parsing.standardise(&mut schema) {
             diagnostics.append(&mut err);
         }
 
+        // Phase 5 formatting: Consistency fixes. These only run during formatting.
+        if transform {
+            if let Err(mut err) = self.standardiser_for_formatting.standardise(ast_schema, &mut schema) {
+                diagnostics.append(&mut err);
```

Comment:
We don't really know what happens when a reformatter-specific validation error happens

Reply:
They mostly get ignored, we'll revisit that later.